CMakeLists.txt

```diff
@@ -71,7 +73,7 @@ endif (OPAE_BUILD_LIBOFS)
 
 option(OPAE_BUILD_SAMPLES "Enable building of OPAE samples" ON)
 mark_as_advanced(OPAE_BUILD_SAMPLES)
-if(OPAE_BUILD_SAMPLES)
+if(OPAE_BUILD_SAMPLES AND OPAE_BUILD_AFU_TEST)
```

Comment:
Isn't this too broad a condition here? Don't we have some samples that are not based off of afu_test?